services/API-service/src/api/event/event.controller.ts

```diff
@@ -174,8 +176,8 @@ export class EventController {
   public async setTrigger(
     @UserDecorator('userId') userId: string,
     @Body() eventPlaceCodesDto: EventPlaceCodesDto,
-  ): Promise<void> {
-    await this.eventService.setTrigger(userId, eventPlaceCodesDto);
+  ): Promise<UpdateResult> {
```

Comment:
I guess this made sense, after functionally testing the just merged PR on ibf-test. Agree?

Reply:
I agree